default/python/AI/ExpansionPlans/ExpansionPlansInterface.py

```diff
@@ -0,0 +1,33 @@
+from collections import OrderedDict
+from typing import FrozenSet, Iterable, Tuple
+
+from common.fo_typing import PlanetId, SpeciesName
+from freeorion_tools.caching import cache_for_current_turn
+
+_the_planner = None
```

Comment:
Planer is a stateless object, so we could assign it here.  

_the_planner = ExpansionPlanner()

In that case, we could remove init. 

Anyway, I prefer lazy initialization. See samples in the empire modules: https://github.com/freeorion/freeorion/tree/master/default/python/AI/empire#empire-info-module

Reply:
I separated the implementaton to avoid a circular include. The implementation include FleetUtilsAI, which indirectly includes expansion_plans.

Lazy initialisation requires that the interface includes the implementation, I cannot break a circular includes with it, can I?

Maybe when we disentangled all this AI spaghetti, we can simplify this module.

Reply:
Circular includes is a problem. Maybe add a comment about it?